README.md

```diff
@@ -864,6 +864,24 @@ In case of your app makes use of its own custom URL scheme for other purpose rat
 
 > In such case you might end up using `my_custom_scheme_tipsi://callback` as return URL and setting `CUSTOM_SCHEME` equals to `my_custom_scheme_tipsi`, following the previous example.
 
+You also need to add into your application's manifest section with redirect activity:
+```xml
+<activity android:name=".RedirectUriReceiver">
```

Comment:
Why I have to add this to my app's manifest file since the same thing is already at `tipsi/android/src/main/AndroidManifest.xml`? Or am I missing something? 💃

Reply:
It's for replacing default value while manifest merging.
I've add default placeholder to libraries manifest. So, for now: if you dont want alipay feature, you don't need to do something manually.